packages/ui-extensions/src/surfaces/customer-account/components/ImageGroup/ImageGroup.ts

```diff
@@ -0,0 +1,48 @@
+import {createRemoteComponent} from '@remote-ui/core';
+import type {RemoteFragment} from '@remote-ui/core';
+
+export interface ImageGroupProps {
+  /**
+   * The layout of the images.
+   * When set to "grid", the images will be displayed in a grid layout with max images of 4.
+   * When set to "inline-stack", the images will be displayed in a single row with max images of 3.
+   *
+   * @default "grid"
+   */
+  variant?: 'grid' | 'inline-stack';
+
+  /**
+   * Accessibility label for the image group.
+   */
+  imagesAccessibilityLabel?: string;
+
+  /**
+   * Loading state of the component.
+   * @default false
+   */
+  loading?: boolean;
+
+  /**
+   * Images to be displayed.
+   *
+   */
+  images: RemoteFragment;
+
+  /**
+   * Total count of items.
+   */
+  totalItems?: number;
+
+  /**
+   * Label for the total count indicator.
+   @default { singular: "item", plural: "items" }
+  */
+  itemName?: {
+    singular: string;
+    plural: string;
+  };
```

Comment:
As per [this discussion](https://github.com/Shopify/core-issues/issues/60848#issuecomment-1779109291), we're going to remove this from the API